samples/framework/libraries/migrate-library-csproj/src/Car/Car.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net45;net451;netstandard1.6</TargetFrameworks>
```

Comment:
How about net452 (still supported), net46, netstandard1.6? ... net45 falls out of the security update window.